src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -136,14 +136,16 @@ const StatusField: FC<StatusFieldProps> = ({ documentation }) => {
   }
   const status = documentation.status;
   const rawDate = documentation.resource?.meta?.lastUpdated;
-  if (rawDate)
+  if (rawDate) {
+    const descript = new Date(rawDate).toLocaleString('en-us');
```

Comment:
Can we rename this to date just so its clear down below
```suggestion
    const date = new Date(rawDate).toLocaleString('en-us');
```